lib/floe/workflow/runner.rb

```diff
@@ -5,29 +5,42 @@ class Workflow
     class Runner
       include Logging
 
-      TYPES         = %w[docker podman kubernetes].freeze
       OUTPUT_MARKER = "__FLOE_OUTPUT__\n"
 
       def initialize(_options = {})
       end
 
+      @runners = {}
       class << self
-        attr_writer :docker_runner
+        # deprecated -- use Floe.set_runner instead
+        def docker_runner=(value)
+          set_runner("docker", value)
+        end
 
-        def docker_runner
-          @docker_runner ||= Floe::Workflow::Runner::Docker.new
+        # see Floe.set_runner
```

Comment:
yard supports see references directly ([docs](https://rubydoc.info/gems/yard/file/docs/Tags.md#see))

```suggestion
        # @see Floe.set_runner
```